CHANGES.rst

```diff
@@ -5,6 +5,15 @@ Changes
 .. ----------
 .. -
 
+1.4.0 (2018-11-19)
+------------------
+
+- new click-odoo-update which implements the functionality of click-odoo-update
```

Comment:
s/functionality of click-odoo-update/functionality of click-odoo-upgrade

Reply:
@sbidoul Are you sure of your description? Shouldn't it be  "new click-odoo-update which implements the functionality of **click-odoo-upgrade** natively ..."